src/app/components/game/game.component.ts

```diff
@@ -164,9 +175,9 @@ export class GameComponent implements OnInit, OnChanges {
                     this.CURSOR_DATA.mouseUpOn = overSq;
                     if (
                         this.CURSOR_DATA.mouseDownOn.x ===
-                            this.CURSOR_DATA.mouseUpOn.x &&
+                        this.CURSOR_DATA.mouseUpOn.x &&
```

Comment:
Did you make this change manually? This is a linting issue, that line SHOULD be indented.  If you didn't make the change manually, your linter is setup wrong, I can help you fix this.  But this is not a change we want to the code. (there are many examples of this throughout your changeset)

Reply:
Yep it's not set up. There are too many stylistic issues that need to be fixed. Probably with a single click.

Reply:
Called with Ali - set up his linter (all looks good now).